frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -251,7 +251,12 @@ export const ADMIN_SETTINGS_SECTIONS = {
         display_name: t`SMTP Security`,
         description: null,
         type: "radio",
-        options: { none: "None", ssl: "SSL", tls: "TLS", starttls: "STARTTLS" },
+        options: [
+          { value: "none", name: "None" },
+          { value: "ssl", name: "SSL" },
+          { value: "tls", name: "TLS" },
+          { value: "starttls", name: "STARTTLS" },
+        ],
```

Comment:
this format violated our types and was inconsistent with every other options key 🙄